### PR TITLE
CA-150200: EN: Duplicate hotkeys for “Storage” and “Show Columns” under Citrix XenCenter.

### PR DESCRIPTION
**Pull request by mcintyre94**
New hotkey for "Show columns" is alt+C

Signed-off-by: Callum McIntyre <callumiandavid.mcintyre@citrix.com>

### CODE REVIEW

**Comment by GaborApatiNagy** [comment]
Looks good

File: XenAdmin/Controls/XenSearch/SearchOutput.resx [hunk 1]
```diff
@@ -218,6 +218,7 @@
   </data>
   <data name="buttonColumns.Text" xml:space="preserve">
     <value>Sh&amp;ow Columns</value>
```
Comment (MihaelaStoica) [inline XenAdmin/Controls/XenSearch/SearchOutput.resx]: Shouln't we remove the the old value?

Comment (mcintyre94) [inline XenAdmin/Controls/XenSearch/SearchOutput.resx]: Oops you're right, I think that's from cherry picking the line in the diff - I'll remove the old value

Comment (mcintyre94) [inline XenAdmin/Controls/XenSearch/SearchOutput.resx]: Fixed in #1100

**Pull request merged**